synthwave84.css

```diff
@@ -114,3 +114,34 @@
   filter: drop-shadow(0 0 5px #03edf9);
 }
+
+.monaco-editor::before {
+    --grid-color:rgba(115,59,139,0.1);
+    --grid-size:60px;
+    --grid-blur:0px;
+  content: '';
+  bottom: 0%;
+  left:0;
+  position:fixed;
+  overflow: visible;
+  margin-left: -50%;
+  width: 200%;
+
+  height: 100%;
+  background-position-y: 0px;
+  background-image:
+    repeating-linear-gradient( 90deg,var(--grid-color) 0%, transparent calc(1px + var(--grid-blur)), transparent var(--grid-size),var(--grid-color) calc(var(--grid-size) + 1px + var(--grid-blur)) ),
+    repeating-linear-gradient( 180deg,var(--grid-color) 0%,transparent calc(1px + var(--grid-blur)), transparent var(--grid-size),var(--grid-color) calc(var(--grid-size) + 1px + var(--grid-blur)) );
+  transform:perspective( 50vh ) rotateX(70deg) translateZ(10px);
+  animation:moving-grid 0.5s infinite linear;
```

Comment:
```suggestion
  animation:moving-grid 10s infinite linear;
```
`0.5` animation speed is too fast and can be annoying, i think that 10s is a nice value :wink: